src/lib/utilities/route-for-api.ts

```diff
@@ -80,6 +86,44 @@ const encode = (parameters: APIRouteParameters): APIRouteParameters => {
   );
 };
 
+export function pathForApi(
```

Comment:
I know I wrote this originally, but it feels like it's getting out of control. 🤣